policyengine_us/variables/gov/states/id/tax/income/deductions/id_aged_or_disabled_deduction.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class id_aged_or_disabled_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho aged or disabled deduction"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        age = person("age", period)
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+        disabled = person("is_disabled", period)
```

Comment:
To make it more sequential.
```suggestion
        p = parameters(
            period
        ).gov.states.id.tax.income.deductions.aged_or_disabled
        # Aged-eligible individuals cannot be head or spouse.
        aged_eligible = (age >= p.age_eligibility) & ~head_or_spouse
        # Disabled-eligible individuals can include head and spouse.
        disabled_eligible = person("is_disabled", period)
```